DigitalLearningSolutions.Data/Services/CustomPromptsService.cs

```diff
@@ -156,6 +157,17 @@ public string GetPromptNameForCentreAndPromptNumber(int centreId, int promptNumb
             return customPromptsDataService.GetPromptNameForCentreAndPromptNumber(centreId, promptNumber);
         }
 
+        public CourseCustomPrompts? GetCustomPromptsForCourse(int customisationId, int centreId, int categoryId)
+        {
+            var result = customPromptsDataService.GetCourseCustomPrompts(customisationId, centreId, categoryId);
+            return result == null ? null :
+                new CourseCustomPrompts(
+                result.CustomisationId,
+                result.CentreId,

```

Comment:
Do we need the customisationID and centre id on the result if these are the params we specify when fetching them?